frontend/public/components/app.jsx

```diff
@@ -270,7 +271,7 @@ const CaptureTelemetry = React.memo(() => {
   const fireTelemetryEvent = useTelemetry();
 
   // notify of identity change
-  const user = useSelector(({ UI }) => UI.get('user'));
+  const user = useSelector(getUser) || {};
```

Comment:
You added functionality here... any reason why the default?

Reply:
not needed, updated.